js/renovation/ui/common/widget.tsx

```diff
@@ -315,6 +316,19 @@ export class Widget extends JSXComponent(WidgetProps) {
     return undefined;
   }
 
+  @Effect() windowResizeEffect(): EffectReturn {
+    const { onDimensionChanged } = this.props;
+
+    if (onDimensionChanged) {
+      const resizeCallback = (): void => onDimensionChanged();
```

Comment:
Can you use onDimensionChanged directly instead of resizeCallback?

Reply:
Yes, I can. Fixed in https://github.com/DevExpress/DevExtreme/pull/16330/commits/fcb3408ec98d8bb1cbf5cac06409c49357272be7